src/lib/databaseUtils.ts

```diff
@@ -1,11 +1,5 @@
 import { Knex } from "knex";
-import {
-  MddbFile,
-  MddbTag,
-  MddbLink,
-  MddbFileTag,
-  File,
-} from "./schema.js";
+import { MddbFile, MddbTag, MddbLink, MddbFileTag, File } from "./schema.js";
```

Comment:
For me this "linting" stuff creates noise in the PRs. I strongly suggest we do this separately (it can just be a direct push to main if just linting).

Reply:
Sure thing. The only thing that makes me lint on PR is our `ci/cd` complaining, but I will select the Github option to bypass this error.

![image](https://github.com/datopian/markdowndb/assets/67981832/4226edaa-f75d-4db5-a93b-18f9924ca486)

Reply:
@mohamedsalem401 👍

For future: we can lint the whole repo once and then we don't get running into these.